src/main/java/uk/gov/pay/ledger/transaction/search/model/SettlementSummary.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.pay.ledger.transaction.search.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+import static uk.gov.pay.commons.model.ApiResponseDateTimeFormatter.ISO_INSTANT_MILLISECOND_PRECISION;
+
+public class SettlementSummary {
+    private ZonedDateTime captureSubmitTime, capturedTime;
+
+    public SettlementSummary(ZonedDateTime settlementSubmittedTime, ZonedDateTime settledTime) {
+        captureSubmitTime = settlementSubmittedTime;
```

Comment:
Is there a reason these have different names, are they equivalent? It might be clearer to have standard naming and then explicitly rename in the `@JsonProperty` or something